libs/components/split-view/src/lib/modules/split-view/split-view-media-query.service.ts

```diff
@@ -31,7 +31,7 @@ export class SkySplitViewMediaQueryService {
     });
   }
 
-  public setBreakpointForWidth(width: number): void {
+  public setBreakpointForWidth(width: number | undefined): void {
```

Comment:
I'm curious why the `| undefined` is needed for the `width` parameter? I removed it here (and in the method below) and the build/test still worked.

Reply:
Probably an artifact of refactoring when an undefined value could have been passed to this method.

Reply:
Fixed.